docs/source-no-noun-replacement-params.md

```diff
@@ -0,0 +1,69 @@
+# source-no-noun-replacement-params
+
+Ensure that source strings do not contain replacement parameters where nouns, noun
+phrases, or adjectives are substituted in to the string. The reason is that the
+spelling of the nouns and adjectives can vary with the linguistic case of that
+part of the sentence. Also, the spelling of the translations of the definite and
+indefinite articles "a", "an", and "the" varies with the gender and plurality of
+the noun or noun phrase being substituted in.
+
+Examples of bad source strings:
+
+```
+Delete the {fileType}.
+Would you like to create a new {fileType} file?
+The {object} has been sent successfully to your company's admin for review.
+```
+
+Very often, the noun, noun phrase or adjective being substituted in has a
```

Comment:
Maybe split off the detailed writeup section with the deep dive into nuances - it's slightly intimidating :)